### PR TITLE
Build project requirements

### PR DESCRIPTION
**Pull request by droberts-sea**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# BackTREK
+
+## Introduction
+
+For this project we'll be returning to the TREK API, using Backbone to build an application that can handle data in complex ways.
```
Comment (kariabancroft) [inline README.md:5]: Could you add the project stage?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## Learning Goals
+
+- Use Backbone and jQuery to work with a complex API
+- Manage application data using Backbone models and collections
+- Build an attractive, robust, and feature-rich user interface
+
```
Comment (kariabancroft) [inline README.md]: Should we add utilizing underscore templates?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+- A user can reserve a spot on a trip
+  - Question: is Backbone API integration useful here? How would this work? What other options are there?
+- The code you write should obey the following rules:
+  - Use Backbone's event workflow. Code that responds to DOM events should be separate from code that updates the DOM.
+  - Any dynamic elements on the page shall be rendered using Underscore templates
+  - Any errors encountered while interacting with the API shall be politely reported to the user
+
+You do not need to implement sorting or filtering yet.
+
+## Wave 2
```
Comment (kariabancroft) [inline README.md]: There are two waves to the project but I don't see the waves being assigned separately. Are we providing both waves at the beginning of the project time?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+- Manage application data using Backbone models and collections
+- Build an attractive, robust, and feature-rich user interface
+
+## Wave 1
+
+In wave 1, you'll use our new functionality (Underscore templates, Backbone models and collections) to achieve feature parity with the original TREK project, including some of the optionals:
+
+- A user can click a link to see a list of trips
+- A user can click on a trip to see details of that trip
+- A user can create a new trip
```
Comment (kariabancroft) [inline README.md]: I guess I haven't seen in the notes yet where we'll be showing them how to do this. Will this be part of the events on collections material?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+#### Normal View
+
+![Normal View](images/wireframe.jpg)
+
+#### Add-Trip Modal
+
+Foundation includes modal functionality, but it can be tricky to get it to work right. Instead, [roll your own](https://www.w3schools.com/howto/howto_css_modals.asp)!
+
+![Add Trip Modal](images/wireframe-modal.png)
```
Comment (kariabancroft) [inline README.md:90]: Thanks for adding these

**Review by kariabancroft — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (droberts-sea) [reply README.md]: By "this", do you mean creating a new trip? That should be covered as the first part of the events lecture.

https://github.com/Ada-Developers-Academy/textbook-curriculum/pull/339/files#diff-5966e4abc47bfbcdc66fe5c172075363

File: README.md (hunk 3 above)
Comment (droberts-sea) [reply README.md]: When I put this together, I was thinking about this project being assigned on Monday, where they don't yet have the tools to do sorting or filtering. Now that we're doing MixCD on Monday and this is on Tuesday, I'm not sure the separation is necessary.

Maybe I'll break it up into:
- Displaying trips
- POSTing data (new trip, reservation)
- Managing the collection

I do think we'll assign all three at once, and students can pace themselves.

File: README.md (hunk 4 above)
Comment (kariabancroft) [reply README.md]: Ok - thanks

**Pull request merged**